csharp/actions/create-extractor-pack/action.yml

```diff
@@ -1,5 +1,9 @@
 name: Build C# CodeQL pack
 description: Builds the C# CodeQL pack
+
+# permissions:
+#   security-events: read
+
```

Comment:
Nit: could you remove this?

Reply:
The reason I put it there is that it's really helpful to be able to look at actions to understand what they require. (And I _think_ that's what that action requires.)

But, I'll remove it.

Reply:
Oh, I see! That's a good idea. Do other Actions use this method to declare what they require? I don't see that we have it in the CodeQL Action, for instance. If it's not a known standard, how about:

```
# Requires `security-events: read` permissions` 
```
or something along those lines?

Reply:
The _better_ behaved repositories w/ a single top level action.yml declare their required permissions in the README.md at the top level.

For example [`actions/stale`](https://github.com/actions/stale?tab=readme-ov-file#recommended-permissions) has a section for it (standard disclaimer: I hate the action).

For repositories that have multiple actions, it's harder -- ideally they'd use README.md files to document how to use the actions, but creating such a file seemed a bit much.

Reply:
That makes sense — thanks for sharing. I'll keep it in mind as it relates to the CodeQL Action, and re-approving here!